clients/desktop/src/vault/balance/visibility/useIsVaultBalanceVisible.ts

```
@@ -1,7 +1,5 @@
-import {
-  PersistentStateKey,
-  usePersistentState,
-} from '../../../state/persistentState'
+import { PersistentStateKey } from '@core/ui/state/PersistentStateKey'
+import { usePersistentState } from '../../../state/persistentState'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)